config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   root "home#index"
+
+  get "/form/new", to: "forms#new"
+  post "/form/new", to: "forms#create"
```

Comment:
Out of interest, is there a reason for setting these routes manually rather than using 
`resources :forms, only: [:new, :create]`?

Reply:
hah, no particular reason, just ended up how i wrote it as I went 😄 can always change it as we go forward